projects/vdk-plugins/airflow-provider-vdk/tests/hooks/test_vdkhook.py

```diff
@@ -1,9 +1,48 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import logging
+import unittest
+from unittest import mock
+
+import requests_mock
 from vdk_provider.hooks.vdk import VDKHook
 
 
-def test_dummy():
-    hook = VDKHook("conn_id", "job_name", "team_name")
+log = logging.getLogger(__name__)
+
+
+# Mock the `conn_sample` Airflow connection
+@mock.patch.dict(
+    "os.environ", AIRFLOW_CONN_CONN_VDK="http://https%3A%2F%2Fwww.vdk-endpoint.org"
+)
+class TestVDKHook(unittest.TestCase):
+    def setUp(self):
+        self.hook = VDKHook(
+            conn_id="conn_vdk", job_name="test_job", team_name="test_team"
+        )
+
+    @requests_mock.mock()
+    def test_start_job_execution(self, m):
+        request_url = "https://www.vdk-endpoint.org/data-jobs/for-team/test_team/jobs/test_job/deployments/production/executions"
+        # Mock endpoint
+        m.post(request_url)
+
+        self.hook.start_job_execution()
```

Comment:
Maybe I'm missing something, but I don't see anything related to the job execution being asserted in this test case. Is there something missing from the diff? (question applies to both test cases)

Reply:
The hook method makes a call through the requests library. This request is picked up by the requests mock object and checked for correctness in the assert calls.

Reply:
Gotcha, thanks